tests/qiskit_convert_test.py

```diff
@@ -1193,6 +1193,63 @@ def test_nonregister_bits() -> None:
         tk_to_qiskit(c)
 
 
+def test_ifelseop_two_branches() -> None:
+    qubits = QuantumRegister(2)
+    clbits = ClassicalRegister(2)
+    circuit = QuantumCircuit(qubits, clbits)
+    (q0, q1) = qubits
+    (c0, c1) = clbits
+
+    circuit.h(q0)
+    circuit.measure(q0, c0)
+
+    with circuit.if_test((c0, 1)) as else_:
+        circuit.h(q1)
+    with else_:
+        circuit.x(q1)
+    circuit.measure(q1, c1)
+
+    tkc = qiskit_to_tk(circuit)
+    assert tkc.n_gates_of_type(OpType.Conditional) == 2
```

Comment:
Should test this more thoroughly. Annoyingly ran into this bug -> https://github.com/CQCL/tket/issues/1726 when trying to validate the contents of the two `CircBox`es.